projects/plugins/crm/includes/ZeroBSCRM.Core.php

```diff
@@ -912,6 +912,7 @@ private function setupUrlsSlugsEtc() {
 		$this->urls['apidocs']           = 'https://automattic.github.io/jetpack-crm-api-docs/';
 		$this->urls['oauthdocs']         = 'https://kb.jetpackcrm.com/knowledge-base/using-gmail-with-jetpack-crm-mail-delivery-system/#setting-up-gmail-oauth-connection-and-mail-delivery-method';
 		$this->urls['woosync']           = 'https://jetpackcrm.com/woocommerce/';
+		$this->urls['woomanagingorders'] = 'https://woocommerce.com/document/managing-orders/';
```

Comment:
Let's use this to jump slightly down the page:

```suggestion
		$this->urls['woomanagingorders'] = 'https://woocommerce.com/document/managing-orders/#order-statuses';
```